README.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```
+clearfix is a way for an element to automatically clear its child elements.  It can be used when an image is taller than it's parent container and thus overflowing outside of the container.
 ```
```

Comment:
I think you're really close. It's not so important the size of the image. What's important is if the image is floated and the parent is not. The parent will collapse, so you have to clearfix in order for that to not happen.